src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,111 @@
+package duke.command;
+
+import duke.TaskList;
+import duke.Ui;
+import duke.Storage;
+import duke.exception.DescriptionFormatException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDos;
+import java.io.IOException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+public class AddCommand extends Command {
+    private String com;
+    private static final DateTimeFormatter DATE_FORMAT_INP = DateTimeFormatter.ofPattern("yyyy-MM-dd' 'HH:mm");
+    private static final DateTimeFormatter DATE_FORMAT_OUT = DateTimeFormatter.ofPattern("MMM dd yyyy HH:mm");
+
+    /**
+     * Constructor for AddCommand.
+     *
+     * @param c command in String format.
+     */
+    public AddCommand(String c) {
+        super(0);
+        com = c;
+    }
+
+    @Override
+    public void execute(TaskList tL, Ui ui, Storage st) {
+        String[] inputs = com.split(" ");
```

Comment:
Perhaps consider name to names because your array contains multiple names. There are a few other arrays you have in other parts of your code that should have a plural name.